forum/tests/test_views.py

```diff
@@ -51,6 +65,7 @@ def test_404(self):
 
     def test_200(self):
         comment = CommentFactory()
+        #  How can this work without passing any variables...or you did above...let me refactor my stuff
         url = comment.get_absolute_url()
```

Comment:
Go back and check the definition of Comment `get_absolute_url`.

Reply:
I see.  Its built into the method.  also just did some more reading on the difference between a function and a method!  should be easier to remember for me